### ISSUE TITLE
CocoaPods Support

### ISSUE THREAD

**Issue by RubenSandwich** [ConversantLabs/SayKitSDK#5]
It would be beneficial to have CocoaPods support on SayKit. 

Not only for the ease of installation but also for the reason of dependency management.

Some problems that would be fixed with CocoaPods:
1. Identifying with version of the SDK you are using
2. Isolation of dependencies
3. Access to past versions

**Comment by gregarious** [comment]
Yep, we'll definitely be rolling out SayKit on CocoaPods -- we'll improve our distribution methods by the time we're ready to publicly release 1.0.
